File: WebApi/Models/Credit.cs
namespace WebApi.Models
{
    public class Credit : BaseEntity
    {
        public string DateDebut { get; set; } = string.Empty;
        public string DateFin { get; set; } = string.Empty;
        public decimal Montant { get; set; }
        public decimal Interets { get; set; }
        public int MembreId { get; set; } 
        public Membre Membre { get; set; } = new Membre{};
        public ICollection<EcheanceCredit> EcheanceCredits { get; set; } = new List<EcheanceCredit>();
    }
}

File: mef/src/app/model/credit.ts
import { EcheanceCredit } from "./echeanceCredit";

export class Credit {
  id: number|null = null;
  dateDebut: string|null = null;
  dateFin: string|null = null;
  capital: number|null = null;
  interet: number|null = null;
  commission: number|null = null;
  echeanceCredits: EcheanceCredit[] = [];
}
